lib/util.js

```diff
@@ -194,6 +194,7 @@ function inspect(obj, opts) {
   if (ctx.colors) ctx.stylize = stylizeWithColor;
   if (ctx.maxArrayLength === undefined) ctx.maxArrayLength = kDefaultMaxLength;
   if (ctx.maxArrayLength === null) ctx.maxArrayLength = Infinity;
+  if (ctx.breakLength === undefined) ctx.breakLength = 60;
```

Comment:
I think `60` is fine, but if set to `null` or `Infinity` explicitly we should figure out what the behavior should be.